{
  "comments": [
    {
      "key": {
        "uuid": "da33351e_588acfbf",
        "filename": "3rd_party/collectd-ves-app/ves_app/ves_app.py",
        "patchSetId": 1
      },
      "lineNbr": 528,
      "author": {
        "id": 304
      },
      "writtenOn": "2017-09-12T15:16:28Z",
      "side": 1,
      "message": "is this intended? it\u0027s reverting the VES App naming in this file",
      "range": {
        "startLine": 528,
        "startChar": 6,
        "endLine": 528,
        "endChar": 15
      },
      "revId": "938c96debc6dc06e821d6c788e4c6df823e16b4a",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}